Beyond08-SOW.md

```diff
@@ -0,0 +1,473 @@
+
+Each team will have the following mentors:
+
+* Team 1: Nati Friedman, TBD 
+* Team 2: Kobi Hakimi, TBD 
+* Team 3: Omer Amsalem, TBD
+* Team 4: Avi Biton, TBD
+
+## Project ideas
+
+You can pick from the ideas below or think up your own.
+
+* Platform for freelancers to offer their services
+* P2P Lending service
+* Dating website
+* Drop shipping web store - selling other people's products on your website
+* Platform for finding part-time jobs for students
+* Personal password maintenance application
+* Course certificate generation and distribution platform
+* An online riddle or math game
+
+## Syllabus (list of sessions)
+
+| #      | Date                  | Topic(s)                            | Lecturer       | Location |
+|--------|-----------------------|-------------------------------------|----------------|----------|
+| -      | Wed, Oct 19, 2022     | First challenge hand-off and student selection| Barak Korren   |          |
+| 1      | Sun, Oct 23, 2022     | Project Architecture overview.      | Barak Korren   |          |
+| **-**  | **Fri, Oct 28, 2022** | **First challenge submission date** | _(Self work)_  |          |
+| 2      | Sun, Oct 30, 2022     | Introduction to Python              | Kobi Hakimi    |          |
+| -      |                       | Second challenge hand-off (~30m session)| Barak Korren   |          |
+| **-**  | **Fri, Nov 04, 2022** | **Second challenge submission date**| _(Self work)_  |          |
+| 3      | Sun, Nov 06, 2022     | Databases, data model and the ORM   | Omer Amsalem   |          |
+|        |                       | Project teams announcement          |                |          |
+| **4**  | **Sun, Nov 13, 2022** | **Demo 1: Development environment** | -              |          |
+| 5      | Sun, Nov 20, 2022     | Business logic and Testing          | Eddy Haas      |          |
+| 6      | Sun, Nov 27, 2022     | _(Open slot)_                       |                |          |
+| **7**  | **Sun, Dec 04, 2022** | **Demo 2: Data Model and business logic**| -              |          |
+| -      | Dec 05 - Dec 08       | Performance review week             | -              |          |
+| 8      | Sun, Dec 11, 2022     | UI: Routes, Views and Templates     | Yariv Rachmani |          |
+| 9      | Sun, Dec 18, 2022     | Frontend, FE, Intro: FE tecnologies | Benny Powers   |          |
+| -      | Sun, Dec 25, 2022     | _(College is closed)_               | -              |          |
+| **10** | **Sun, Jan 01, 2023** | **Demo 3: UI and user workflow**    | -              |          |
+| 11     | Sun, Jan 08, 2023     | Going to production                 | Benny Powers   |          |
+| 12     | Sun, Jan 15, 2022     | AMA - Ask Me Anything, Software engineer in the industry| _(all mentors)_|          |
+| **13** | **Sun, Jan 22, 2022** | **Final Demo: demo & retrospective**| -              |          |
```

Comment:
I think this is the location that should be 
@ifireball could you please verify it

```suggestion
|--------|-----------------------|-------------------------------------|----------------|----------|
| -      | Wed, Oct 19, 2022     | First challenge hand-off and student selection| Barak Korren   |    Tel Hai Collage      |
| 1      | Sun, Oct 23, 2022     | Project Architecture overview.      | Barak Korren   |     Video Meeting     |
| **-**  | **Fri, Oct 28, 2022** | **First challenge submission date** | _(Self work)_  |          |
| 2      | Sun, Oct 30, 2022     | Introduction to Python              | Kobi Hakimi    |     Video Meeting     |
| -      |                       | Second challenge hand-off (~30m session)| Barak Korren   |          |
| **-**  | **Fri, Nov 04, 2022** | **Second challenge submission date**| _(Self work)_  |          |
| 3      | Sun, Nov 06, 2022     | Databases, data model and the ORM   | Omer Amsalem   |     Video Meeting     |
|        |                       | Project teams announcement          |                |          |
| **4**  | **Sun, Nov 13, 2022** | **Demo 1: Development environment** | -              |     Video Meeting     |
| 5      | Sun, Nov 20, 2022     | Business logic and Testing          | Eddy Haas      |     Video Meeting     |
| 6      | Sun, Nov 27, 2022     | _(Open slot)_                       |                |     Video Meeting     |
| **7**  | **Sun, Dec 04, 2022** | **Demo 2: Data Model and business logic**| -              |     Video Meeting     |
| -      | Dec 05 - Dec 08       | Performance review week             | -              |     Video Meeting     |
| 8      | Sun, Dec 11, 2022     | UI: Routes, Views and Templates     | Yariv Rachmani |     Video Meeting     |
| 9      | Sun, Dec 18, 2022     | Frontend, FE, Intro: FE tecnologies | Benny Powers   |    Video Meeting      |
| -      | Sun, Dec 25, 2022     | _(College is closed)_               | -              |          |
| **10** | **Sun, Jan 01, 2023** | **Demo 3: UI and user workflow**    | -              |  Video Meeting        |
| 11     | Sun, Jan 08, 2023     | Going to production                 | Benny Powers   |    Video Meeting      |
| 12     | Sun, Jan 15, 2022     | AMA - Ask Me Anything, Software engineer in the industry| _(all mentors)_|    Red Hat office(Gamla)      |
| **13** | **Sun, Jan 22, 2022** | **Final Demo: demo & retrospective**| -              |    Margalit Center      |
```